src/fparser/parsefortran.py

```diff
@@ -234,7 +233,7 @@ def simple_main():
         return parse_all_f()
     for filename in sys.argv[1:]:
         reader = FortranFileReader(filename)
-        print(yellow_text('Processing '+filename+' (mode=%r)' % (reader.mode)))
+        print('Processing '+filename+' (mode=%r)' % (reader.mode))
```

Comment:
This line is not covered. In fact, is this routine (`simple_main()`) ever actually used anywhere?

Reply:
It's called from `if __name__ == "__main__":` just below

Reply:
This doesn't seem appropriate for unit testing. This seems like a system testing problem.

Reply:
Agreed. I'm not sure we even want to keep the ability to 'run' parsefortran.py directly but that's something for another issue.